server/models/user.model.js

```diff
@@ -208,6 +208,74 @@ const UserSchema = new mongoose.Schema(
       website: {
         type: String,
       },
+      verification: {
+        companyInfo: {
+          comments: [
+            {
+              comment: {
+                type: String,
+              },
+              addedBy: {
+                type: ObjectId,
+              },
+            },
+          ],
+          status: {
+            type: String,
+            default: VENDOR_INFO_STATUS.IN_REVIEW,
```

Comment:
default should be PENDING 🙏 
```suggestion
            default: VENDOR_INFO_STATUS.PENDING,
```

Here is the breakdown:

```
PENDING: 'Pending',   -> We are expecting information from the user
IN_REVIEW: 'In Review', -> The user has submitted info, we are currently reviewing it. If a comment is added, that this should go back to PENDING
VERIFIED: 'Verified', -> We have approved the Step.
```